src/CoreText/Adapter.cs

```diff
@@ -140,6 +153,18 @@ public static void SetValue (IDictionary<NSObject, NSObject> dictionary, NSObjec
 				dictionary.Remove (key);
 		}
 
+		public static void SetValue (NSDictionary dictionary, NSObject key, bool? value)
+		{
+			if (value.HasValue) {
+				AssertWritable (dictionary);
+				CFMutableDictionary.SetValue (dictionary.Handle, key.Handle, value.value ? CFBoolean.True.Handle : CFBoolean.False.Handle);
```

Comment:
Doesn't this work?

```csharp
dictionary [key] = new NSNumber (value.Value);
```

that's simpler, and also match the other overloads.

Reply:
@rolfbjarne  Is `new NSNumber (bool)` the same as `CFBoolean.True.Handle` or `CFBoolean.False.Handle`?, the API that uses this wants a `CFBoolean`

Reply:
Yes: `NSNumber is “toll-free bridged” with its Core Foundation counterparts: CFNumber for integer and floating point values, and CFBoolean for Boolean values.` (https://developer.apple.com/documentation/foundation/nsnumber)

Reply:
Ahh good point, forgot about toll-free bridging